src/python/WMArchive/Schemas/FWJRProduction.py

```diff
@@ -74,42 +73,54 @@
                          'validStatus': 'PRODUCTION',
                          "SEName": "srm-cms.cern.ch",
-                         "PNN": "T2_CERN_CH",
+                         "PNN": "T2_CH_CERN",
                          "GUID": '',
                          "StageOutCommand": "srmv2-lcg"}],
-              'performance': {#"multicore": {},
-                  "storage": {
-                    "readAveragekB": 77.8474891246,
-                    "readCachePercentageOps": 0.0,
-                    'readMBSec': 0.0438598972596,
-                    'readMaxMSec': 4832.84,
-                    'readNumOps': 97620.0,
-                    'readPercentageOps': 1.00032780168,
-                    'readTotalMB': 7423.792,    
-                    'readTotalSecs': 0.0,
-                    'writeTotalMB': 357.624,
-                    'writeTotalSecs': 575158.0},
-                    "memory": {
-                       "PeakValueRss": 0.0,
-                       "PeakValueVsize": 0.0
-                    },
-                   "cpu": {
-                       "TotalJobCPU": 0.39894,
-                       "AvgEventCPU": -2.0, # for ("-nan")
-                       "MaxEventCPU": 0.0,
-                       "AvgEventTime": -1.0, # for ("inf")
-                       "MinEventCPU": 0.0,
-                       "TotalEventCPU": 0.0,
-                       "TotalJobTime": 26.4577,
-                       "MinEventTime": 0.0,
-                       "MaxEventTime": 0.0
-                   }},
+              'performance': {'cpu': [{'pName': 'TotalJobCPU',
+                                        'value': 159.08},
+                                       {'pName': 'AvgEventCPU',
+                                        'value': 0.0543207},
+                                       {'pName': 'MaxEventCPU',
+                                        'value': 2.01969},
+                                       {'pName': 'AvgEventTime',
+                                        'value': 0.0915972},
+                                       {'pName': 'MinEventCPU',
+                                        'value': 0.0},
+                                       {'pName': 'TotalEventCPU',
+                                        'value': 128.957},
+                                       {'pName': 'TotalJobTime',
+                                        'value': 217.452},
+                                       {'pName': 'MinEventTime',
+                                        'value': 0.000827074},
+                                       {'pName': 'MaxEventTime',
+                                        'value': 3.28857}],
+                              'memory': [{'pName': 'PeakValueRss',
+                                           'value': 692.926},
+                                          {'pName': 'PeakValueVsize',
+                                           'value': 952.109}],
+                              'multicore': [],
```

Comment:
How about we'll put a comment about potential pName, e.g. #pName values can be writeTotalMB, readPercentageOps, etc.

But I just checked that I can generate schema using either way, i.e. abstract way I suggested or explicit way you provided. If you feel that showing actual names and their values in dict is more appealing than it's fine with me.